src/echo/echo.rs

```diff
@@ -174,8 +174,7 @@ pub fn uumain(args: Vec<String>) -> i32 {
     };
 
     if !free.is_empty() {
-        // NB: Using deprecated connect() until Rust 1.3 becomes stable.
-        let string = free.connect(" ");
+        let string = free.join(" ");
```

Comment:
code doesn't seem to compile on beta or stable - this removes the warnings on Nightly and Beta